### PR TITLE
Robin/#3/refactor components

### PR DESCRIPTION
**Pull request by RobinRz**

### CODE REVIEW

File: src/components/Listviewitem/ListViewItem.jsx [hunk 1]
```diff
@@ -2,20 +2,20 @@ import React from 'react'
...
 const ListViewItem = (props) => {
     return (
-        <div className='flex justify-between w-full bg-white m-4 p-3 rounded-md'>
+        <div className={`flex justify-between w-full bg-white m-4 p-3 rounded-md ${props.className}`}>
             <div className='rounded-full border border-gray-400 w-16 h-16 flex items-center justify-center'>
                 <img src='/#' className='' />
             </div>
             <span>
-            {props.item.item}
+                {props.item.item}
```
Comment (sreeragmsurendran) [inline src/components/Listviewitem/ListViewItem.jsx:10]: unwanted space

Comment (RobinRz) [reply src/components/Listviewitem/ListViewItem.jsx:10]: hehe ath ninte item aan.. yaa i will push in a sec

Comment (RobinRz) [reply src/components/Listviewitem/ListViewItem.jsx:10]: ath prettier format cheythathan. line gap indo

Comment (RobinRz) [reply src/components/Listviewitem/ListViewItem.jsx:10]: mine looks like this:
image

Comment (sreeragmsurendran) [reply src/components/Listviewitem/ListViewItem.jsx:10]: Approved

**Pull request merged**